modules/ducktests/tests/ignitetest/services/utils/control_utility.py

```diff
@@ -248,12 +311,15 @@ def __parse_cluster_state(output):
 
         return ClusterState(state=state, topology_version=topology, baseline=baseline)
 
-    def __run(self, cmd):
-        node = random.choice(self.__alives())
+    def __run(self, cmd, node=None):
+        if node is None:
+            node = random.choice(self.__alives())
 
         self.logger.debug(f"Run command {cmd} on node {node.name}")
 
-        raw_output = node.account.ssh_capture(self.__form_cmd(node, cmd), allow_fail=True)
+        node_ip = socket.gethostbyname(node.account.hostname)
```

Comment:
what is the difference with "node.account.externally_routable_ip" used before?

Reply:
node.account.externally_routable_ip is hostname
for the dump file to be generated on a specific node, we must pass ip

Reply:
>> node.account.externally_routable_ip is hostname
so, what the difference between "hostname" (node.account.externally_routable_ip) and node.account.hostname?

Reply:
example:
node.account.hostname == ducker02
node.account.externally_routable_ip == ducker02

we need to use an ip address: 172.22.0.3